src/main/java/com/exadel/sandbox/controller/EmployeeController.java

```diff
@@ -0,0 +1,53 @@
+package com.exadel.sandbox.controller;
+
+import com.exadel.sandbox.dto.EmployeeDto;
+import com.exadel.sandbox.entities.Employee;
+import com.exadel.sandbox.mapper.EmployeeMapper;
+import com.exadel.sandbox.service.EmployeeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("employee")
+@RequiredArgsConstructor
+public class EmployeeController {
+
+    private final EmployeeService employeeService;
+    private final EmployeeMapper mapper;
+
+    @GetMapping("list")
+    List<EmployeeDto> getEmployees() {
+        List<Employee> employees = employeeService.getEmployees();
+        return employees.stream().map(mapper::toDto).collect(Collectors.toList());
+    }
+
+    @GetMapping("{id}")
+    EmployeeDto getEmployeeById(@PathVariable("id") Integer id) {
+        Employee employee = employeeService.getEmployeeByID(id);
+        System.out.println(employee.getEmId());
+        return mapper.toDto(employee);
+    }
+
+    @PutMapping()
```

Comment:
We read about REST principles. 
In general:
Using POST http method we should create something
Using PUT http method we should update something
Using GET http method we should receive something
Using DELETE http method we should delete something